### DISCUSSION TITLE
Equilux calculation

### DISCUSSION THREAD

**Discussion by PneumaBlessing** [Stellarium/stellarium#2765] [General]
I'm interested in finding the Equilux of certain locations using Stellarium.  Since it has so much of the data and algorithms, I wanted to know if it has a feature where I can enable that or a way to find that information for a specific city.  Again this is for Equilux times and not Equinox.  Specifically the Spring Equilux.

**Comment by alex-w** [comment]
Stellarium known nothing about Equilux. Someone should add the algorithms of computation of equiluxes.

**Comment by gzotti** [comment]
If you can provide a useful reference that has a really clear definition and algorithm to compute it? These neologisms are sometimes mentioned, but everybody might understand something slightly different.

**Comment by PneumaBlessing** [comment]
So what I want to be able to do is get the output of the difference between the time of hours of darkness vs the time of the hours of light based on a specific city or location.  However, I want the darkness time to be the period BEFORE the Light.  For Example:

Let's take March 20th, 31 AD.  What is the difference between the time or period of darkness that occurred on the night of 19 March vs the daylight period that occurred on March 20th.  I would say a comparison of when the sun diameter drops over the horizon at the given location vs when the sun's diameter rises over the horizon and vice versa for the daylight hours.

I'm a believer in an alignment of the certain "classical" planets/stars in relation to Biblical events.  For example, I see March 25 (Julian) as the day of the Crucifixion in 31 AD.  My reasoning takes Moses commands quite specifically, in that I believe that Jesus had to be apprehended on the 10th of the month and I believe that would have started on the night of the 20th of March in 31 AD.  I then believe that Jesus was brought before Pilate on the morrow (daylight of the 21st) which I believe is still the 10th of the lunar month (new moon begins month) and then was possibly held in a pit in the palace of Caiaphas for 3 days and night until crucified on the 14th.  There is weell documented evidence of that pit location on the internet and References from Isaiah that He was to be in a pit.

So this is all based on the Equilux as opposed to the Equinox.  For the period the 10th of the month could only have occurred in 31 AD direclty after the Equilux.  Then crucifixion on the 25th corresponds to what early Church Historians dated as the 25th of March or the 8th of the Kalends of April.

This also by inclusively counting years of the destruction of the Temple at 40 years from the Crucifixion.

I have a theory that I'm trying to confirm (going to take a lot of work) but it is that the Sabbatical years can be confirmed through astronomical methods.  I have a theory on how to confirm this but I don't want to give out that information till I have had a chance to follow it further.  I put the 80th Jubilee as would have been declared in 31 AD.

**Comment by gzotti** [comment]
It seems you also have to take local horizon altitude into account. And of course, after sunset there is still twilight, not immediate darkness. This is why I asked for definitions. 
If it's only one year you are interested in, checking sunrise and sunset at ~5 dates manually should not take more than a few minutes.

**Reply by PneumaBlessing** [reply]
Yes, true.  I the horizon is going to be a mandatory attribute of the algorithm.   But again, I'm saying when the diameter of the sun sinks below that horizon.  I'm not interested in one year but many many years.

**Reply by gzotti** [reply]
There is no automated way to do that, yet.

**Reply by PneumaBlessing** [reply]
I appreciate the feedback. I'll go through manually and try to do my work that route.
